src/styles/semantic-tokens.css

```diff
@@ -88,6 +88,7 @@
     --card-gradient: linear-gradient(123deg, rgba(255, 255, 255, 0.20) 58.99%, rgba(174, 110, 203, 0.13) 104.04%);
     --card-gradient-secondary: linear-gradient(95deg, rgba(211, 145, 242, 0.12) 0%, rgba(159, 43, 212, 0.12) 102.78%);
     --card-gradient-secondary-hover: linear-gradient(95deg, rgba(211, 145, 242, 0.2) 0%, rgba(159, 43, 212, 0.2) 102.78%);
+    --ten-year-gradient: linear-gradient(100deg, #F6C9EA 55.38%, #C7A9F1 110.54%);
```

Comment:
Similar point with this gradient and variable name. Im fine adding it as long as this is a temporary page. Otherwise, would encourage you to create a better name to be reused later in other components.

cc @nloureiro @konopkja would be nice to define all the gradients we have in the DS to have consistency in future interations. [current gradients](https://github.com/ethereum/ethereum-org-website/blob/10-year-anniversary/tailwind.config.ts#L227)